spacepy/irbempy/irbempy.py

```diff
@@ -1258,6 +1260,352 @@ def get_AEP8(energy, loci, model='min', fluxtype='diff', particles='e'):
     return flux[0, 0]
 
 
+class Shieldose2(dm.SpaceData):
+    """
+    A class for performing dose calculations using Shieldose2
+
+    Notes
+    -----
+    .. versionadded:: 0.5.0
+
+    Examples
+    --------
+    >>> import spacepy.irbempy as ib
+    >>> import spacepy.toolbox as tb
```

Comment:
It's a short example where these lines are on the same screen as the locations they're used. I like having shorter lines in the example (it's easier to read for me), and I dislike `from spacepy import toolbox` because at least `tb` is obviously shorthand.
I'll switch this to full names, but I think it looks very clunky.

Reply:
Don't worry about changing it if you disagree; we just need to thrash this out "sometime".